### PR TITLE
Allow calico-node to update packetcaptures

### PR DESCRIPTION
**Pull request by asincu**
## Description

Enhance calico-node clusterrole to update packetures CRs with their status.

## For PR author

- [ ] Tests for change.
- [ ] If changing pkg/apis/, run `make gen-files`
- [ ] If changing versions, run `make gen-versions`

## For PR reviewers

A note for code reviewers - all pull requests must have the following:

- [ ] Milestone set according to targeted release.
- [ ] Appropriate labels:
  - `kind/bug` if this is a bugfix.
  - `kind/enhancement` if this is a a new feature.
  - `enterprise` if this PR applies to Calico Enterprise only.

### CODE REVIEW

**Review by rene-dekker — APPROVED** [review]
lgtm

File: pkg/render/node.go [hunk 1]
```diff
@@ -384,6 +384,14 @@ func (c *nodeComponent) nodeRole() *rbacv1.ClusterRole {
 				},
 				Verbs: []string{"create"},
 			},
+			{
+				// Tigera Secure updates status for packet captures.
+				APIGroups: []string{"crd.projectcalico.org"},
+				Resources: []string{
+					"packetcaptures",
```
Comment (tmjd) [inline pkg/render/node.go:391]: Would `packetcaptures/status` work (like `pod/status` at Line 281)?

Comment (asincu) [reply pkg/render/node.go:391]: Not really. I tried using only packetcapture/status, but it did not work.

Comment (tmjd) [reply pkg/render/node.go:391]: As you pointed out in slack, libcalico-go doesn't provide a Status only update mechanism so I think that is why this isn't possible. Bummer.

**Review by tmjd — APPROVED** [review]
LGTM

**Pull request merged**